src/Administration/Resources/app/administration/src/docs/07-testing/01-overview.md

```diff
@@ -0,0 +1,142 @@
+# Testing Overview
+
+## Introduction
+
+The Shopware Administration uses **Jest** as the primary testing framework for unit and component tests. This document provides a basic overview of the testing infrastructure and common patterns.
+
+For detailed examples and advanced topics, see the [official Shopware Jest testing documentation](https://developer.shopware.com/docs/guides/plugins/plugins/testing/jest-admin.html).
+
+## Technology Stack
+
+- **Jest** - Core testing framework
+- **Vue Test Utils** - Vue.js component testing utilities
+- **@vue/vue3-jest** - Vue 3 component transformer
+- **TypeScript Support** - Full TypeScript support for test files
+
+## Test File Location
+
+Test files are co-located with the components they test:
+
+```text
+src/app/component/base/sw-button/
+├── index.js
+├── sw-button.html.twig
+└── sw-button.spec.js          # Test file
+```
+
+**Naming Convention:** `[component-name].spec.js` or `[component-name].spec.ts`
+
+## Running Tests
+
+```bash
+# Run all tests
+composer run admin:unit
+
+# Watch mode for development
+composer run admin:unit:watch
+
+# Generate component imports (required before first run)
+npm run unit-setup
+```
+
+## Basic Test Structure
+
+### Component Tests
+
```

Comment:
Do we need something special here how to test State/Store?

Reply:
Details can be found in the "Additional Resources"